pkgs/development/ruby-modules/bundled-common/functions.nix

```diff
@@ -80,6 +80,8 @@ in rec {
           outputs = [ "out" ];
           out = res;
           outputName = "out";
+          suffix = version;
+          gemType = "path";
```

Comment:
Should I add a comment mentioning the PR/commit that introduced the need for these attributes?